R/bayes.R

```diff
@@ -247,23 +220,6 @@ stan_coefs <- function(x, y, options, verbose, ...) {
   coefs
 }
 
-
-map_glm_coef <- function(dat, mapping) {
```

Comment:
This function is here: https://github.com/tidymodels/embed/blob/main/R/glm.R#L170-L181